html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -185,6 +185,13 @@ export default {
   mounted () {
     this.$store.dispatch('$_nodes/getNode', this.mac)
     this.$store.dispatch('config/getViolations')
+    var vm = this
+    window.addEventListener('keyup', function (event) {
+      switch (event.keyCode) {
+        case 27: // escape
+          vm.close()
+      }
+    })
```

Comment:
When adding an event listener, you must also remove it when the component is destroyed.

I've pushed a commit (dba8813) to handle the ESC key. You can remove this change.